docs/endpoints/openrtb2/amp.md

```diff
@@ -31,3 +31,19 @@ A sample response payload looks like this:
 
 In [the typical AMP setup](http://prebid.org/dev-docs/show-prebid-ads-on-amp-pages.html),
 these targeting params will be sent to DFP.
+
+### Query Parameters
+
+This endpoint supports the following query parameters:
+
+1. `h` - `amp-ad` `height`
+2. `w` - `amp-ad` `width`
+3. `oh` - `amp-ad` `data-override-height`
+4. `ow` - `amp-ad` `data-override-width`
+5. `ms` - `amp-ad` `data-multi-size`
+6. `curl` - the canonical URL of the page
+7. `purl` - the page URL
+8. `timeout` - the publisher-specified timeout for the RTC callout
```

Comment:
Maybe worth moving `debug=1` into this list too?

I know it's mentioned above... but this list will pop out at people much more than the english paragraphs.